docs/api-gateway-v2/03-04-migration.md

```diff
@@ -0,0 +1,144 @@
+---
+title: Migration from the previous Api resources
+type: Details
+---
+
+The migration is done automatically by a job. During the migration the old Api object is being migrated to an APIRule object, what may cause temporary unavailability of the exposed service. After the successful migration the old resource is removed. The migration might be skipped for some Api configuration, in which case the manual migration process may be done. It won't break the existing service exposure, but any changes to the Api object won't be effective.
```

Comment:
> The migration is done automatically by a job.

I suggest you describe the job and explain when it is triggered.

> During the migration the old Api object is being migrated to an APIRule object, what may cause temporary unavailability of the exposed service.

Avoid repetitions (migration, migrated). The "temporary unavailability" sounds awkward. My suggestion: "During the migration, the old Api object is being translated to an APIRule object, which may result in a temporary downtime of the exposed service.

> After the successful migration the old resource is removed.

"The original resource is deleted as a part of the migration process."

> The migration might be skipped for some Api configuration, in which case the manual migration process may be done.

Let's turn it into a warning, shall we? Then you can provide a link to the relevant section that describes manual migration. 

> It won't break the existing service exposure, but any changes to the Api object won't be effective.

This sentence is confusing and I can't quite figure out what it refers to.

Reply:
this is by no means outdated!